src/components/Molecules/CTAMultiCard/SingleCard.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { snakeCase } from 'lodash';
+import Picture from '../../Atoms/Picture/Picture';
+import ArrowIcon from './ArrowIcon';
+import allowListed from '../../../utils/allowListed';
+import {
+  CardLink,
+  ImageWrapper,
+  CopyAndLinkSection,
+  Copy,
+  CTA,
+  CTAText,
+  ArrowIconWrapper,
+  CardWrapper
+} from './CTAMultiCard.style';
+
+const isInternalUrl = url => allowListed(url) || url.charAt(0) === '/' || url.charAt(0) === '#';
+
+const SingleCard = ({
+  card,
+  isCarousel
+}) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const {
+    id,
+    body,
+    image,
+    backgroundColour,
+    link,
+    linkLabel
+  } = card;
+
+  const fallback = image?.gatsbyImageData?.images?.fallback?.src;
```

Comment:
I'm really sorry I didn't catch this first time around, nor preempt it within the work:

Typically, this is something we'd do within the _CR_ repo (since it's specifically tied to Gatsby and its weirdness) _before_ passing it into the component, letting us make this component as agostic as possible, and use it in other contexts.

I don't know if Curtis has plans to use this within Donate or any other minisite (Giftaid, Contact Us, etc.) we have planned where we'd _need_ it to just take simplified images/objects but I guess we can cross that bridge when we come to it, and refactor this then.

Sorry for the lack communication; happy to leave this aspect as-is for now.